docs/content/preview/architecture/transactions/isolation-levels.md

```diff
@@ -7,7 +7,7 @@ menu:
   preview:
     identifier: architecture-isolation-levels
     parent: architecture-acid-transactions
-    weight: 1152
+    weight: 20
```

Comment:
Wondering if the link title should be changed to "Transaction isolation" just like in the landing page.